### ISSUE TITLE
Email address & subject incorrect in compose window after clicking on reply all/reply arrow

### ISSUE THREAD

**Issue by amourchaleur** [thunderbird-conversations/thunderbird-conversations#1891]
* **Thunderbird version:** 91.12.0
* **Thunderbird Conversation version:** 4.0.22
* **Operating System:** Windows 11

**What did you do?**
After clicking on the reply or reply all arrow circled in red. In the compose window, the email subject and “From” email address is not the correct one. **It shows the email subject and email of the last replied email in my outbox**. Please see attached screenshot for clearer explanation. 

![Screenshot 2022-08-25 113640](https://user-images.githubusercontent.com/16366455/186574150-2b512599-4b96-436b-9285-84b381c90540.png)

Thank you.

**Comment by Standard8** [comment]
Are you sure that header wasn't at the bottom of the original message that you responded to?

I can't find a way that Thunderbird would produce a multi-line original message for reply. You get something similar if you forward a message, but that is a table not simple lines of text.

**Comment by amourchaleur** [comment]
When we click reply, it quoted the original message below, the issue occurs at the quoted message, if you get what I meant. Try clicking the reply or reply all arrow in the conversation, it pop ups a compose reply window which will quote the original message (the issue occurs at this part).

**Comment by Standard8** [comment]
This is what I get when I click reply to a message:

<img width="385" alt="Screenshot 2022-08-25 at 09 56 51" src="https://user-images.githubusercontent.com/520844/186621783-e92caecd-3bfb-4c94-81bd-6fb0bee86376.png">

As you can see, the header there is of format:

`On <date>, <name> wrote:`

Your one is of format:

```
---------- original message ---------------
Subject: 
From:
To:
Date:
```

As far as I can tell, Thunderbird doesn't produce the format shown in your picture. Therefore it was most likely already embedded in the original email that you responded to.

I also have tried several things to try and reproduce this, but I cannot.

**Comment by Standard8** [comment]
My other thought is to try clicking the three-dots at the right-hand end and "View using the Classic Reader", and then try clicking reply from there.

**Comment by amourchaleur** [comment]
> My other thought is to try clicking the three-dots at the right-hand end and "View using the Classic Reader", and then try clicking reply from there.

Tried this and there is no issues with this method though. Works perfect if I click reply from the classic reader.

**Comment by Standard8** [comment]
Does it happen with every message?

Do you have other add-ons installed?

Have you changed some of the advanced preferences?

Conversations calls through to Thunderbird's standard reply functions, and for those it simply passes the message that is being replied to and what identity to use for the reply. Therefore I find it difficult to see how just the subject and name could be wrong.

Which makes me think there are other factors at play - especially given that original message output is not something that Thunderbird supports by default.

If it is only this email that is doing it, try viewing the source of the email and maybe seeing if that code is embedded somewhere.

**Comment by amourchaleur** [comment]
I just realised I have the change quote plugin enabled, once I deactivated it, it went back to the original thunderbird reply quote (like your screenshot).

**Comment by Standard8** [comment]
Thank you for finding that, I have been able to reproduce it with that add-on installed. Unfortunately there's not much I can do from the Conversation's end - the change quote add-on is not handling the reply mechanism correctly, and as a result it uses the last message header all the time.

I've filed [an issue with the add-on directly detailing the findings](https://github.com/caligraf/ChangeQuote/issues/34). I will close this out as there isn't anything else I can do here.
